spec/controllers/hyrax/templates_controller_spec.rb

```diff
@@ -30,7 +30,8 @@
       it 'redirects to index' do
         delete :destroy, params: { id: template.name }
 
-        expect(response).to render_template :index
+        expect(response)
+          .to redirect_to "/templates?locale=en&notice=Deleted+#{template.name}"
```

Comment:
I'm not terribly happy with this, but I couldn't figure out the correct way to get RSpec to find the correct routes for `template_path`, which is on `main_app`. @little9, any ideas?